schema/src/main/scala/CpgExtSchema.scala

```diff
@@ -165,6 +165,37 @@ class CpgExtSchema(builder: SchemaBuilder, cpgSchema: CpgSchema) {
 
   templateDOM.addOutEdge(edge = sourceFile, inNode = file)
 
+  // Nodes and edges for Android res/layout/*.xml files and
+  // Android manifest.xml files for <uses-permission>
+
+  // Android XML Layout Node - Begin
+  val androidLayoutNode = builder
```

Comment:
We had some brainstorming on using existing node vs creating new ones, we found the use of creating new use case specific node better than and easy to maintain if we want to add node specific properties, instead of mapping values to existing properties
Cc - @tuxology